project/CMakeLists.txt

```diff
@@ -183,11 +183,6 @@ if (NOT ELF_INCLUDE_FOUND)
 	message(FATAL_ERROR "Cannot find elf.h. Make sure you have it installed")
 endif()
 
-find_package(LibDwarf QUIET)
```

Comment:
why do you remove libdwarf dependency?

Reply:
So that folly will not build the experimental exception_tracer lib, which causes a lot of problem, such as not working on lower versions of linux

Reply:
Make sense. We only need the cachelib library.